### PR TITLE
Would like to add butter to make even more moist our cake.

### PR DESCRIPTION
**Pull request by catarinareis**

### CODE REVIEW

**Comment by Joao280** [comment]
Like!

File: cake-recipe.txt [hunk 1]
```diff
@@ -16,7 +17,7 @@ Directions:
     pans.
  
  2. In a large bowl, combine the sugar, flour, cocoa, baking powder, baking soda
-    and salt.  Add eggs, milk, oil and vanilla. Mix until smooth.  Stir in
+    and salt.  Add eggs, butter, milk, oil and vanilla. Mix until smooth.  Stir in
```
Comment (Kazhw) [inline cake-recipe.txt]: this is a comment

Comment (Joao280) [inline cake-recipe.txt]: Ok!

Comment (Joao280) [inline cake-recipe.txt]: images

**Pull request merged**